docs/rpc_nodes.md

```diff
@@ -18,15 +18,15 @@ author: Roxane Letourneau
 | Provider         | Net         | URL                                | Header      |  
 |------------------|-------------|------------------------------------|-------|
 | ECAD Labs        | Mainnet     | https://mainnet.api.tez.ie         | [Check](https://mainnet.api.tez.ie/chains/main/blocks/head/header)      |
-| ECAD Labs        | Hangzhounet | https://hangzhounet.api.tez.ie     | [Check](https://hangzhounet.api.tez.ie/chains/main/blocks/head/header)     |
 | ECAD Labs        | Ithacanet   | https://ithacanet.ecadinfra.com    | [Check](https://ithacanet.ecadinfra.com/chains/main/blocks/head/header)      |
+| ECAD Labs        | Jakartanet  | https://jakartanet.ecadinfra.com   | [Check](https://jakartanet.ecadinfra.com/chains/main/blocks/head/header)      |
 | SmartPy          | Mainnet     | https://mainnet.smartpy.io         |  [Check](https://mainnet.smartpy.io/chains/main/blocks/head/header)     |
-| SmartPy          | Hangzhounet | https://hangzhounet.smartpy.io/    | [Check](https://hangzhounet.smartpy.io/chains/main/blocks/head/header)      |
 | SmartPy          | Ithacanet   | https://ithacanet.smartpy.io/      | [Check](https://ithacanet.smartpy.io/chains/main/blocks/head/header)    |
 | Tezos Foundation | Mainnet     | https://rpc.tzbeta.net/            | [Check](https://rpc.tzbeta.net/chains/main/blocks/head/header)      |
-| Tezos Foundation | Ithacanet   | https://rpczero.tzbeta.net/        | [Check](https://rpczero.tzbeta.net/chains/main/blocks/head/header)      |
+| Tezos Foundation | Jakartanet  | https://rpczero.tzbeta.net/        | [Check](https://rpczero.tzbeta.net/chains/main/blocks/head/header)      |
+| LetzBake!        | Mainnet     | https://teznode.letzbake.com       |  [Check](https://teznode.letzbake.com/chains/main/blocks/head/header)     |
```

Comment:
LetzBake should be removed.
Marigold nodes seem to have been removed in the preview.